predictor.py

```diff
@@ -62,6 +62,10 @@ def predict(
         instance_prompt: str = Input(
             description="The prompt you use to describe your training images, in the format: `a [identifier] [class noun]`, where the `[identifier]` should be a rare token. Relatively short sequences with 1-3 letters work the best (e.g. `sks`, `xjy`). `[class noun]` is a coarse class descriptor of the subject (e.g. cat, dog, watch, etc.). For example, your `instance_prompt` can be: `a sks dog`, or with some extra description `a photo of a sks dog`. The trained model will learn to bind a unique identifier with your specific subject in the `instance_data`.",
         ),
+        instance_prompt_use_file: bool = Input(
+            description="Whether to append to instance prompt from file.",
```

Comment:
Drive-by comment (I know this is a WIP): 

Maybe include some examples here in the description to help illustrate the concept? This is effectively the documentation for this feature so it should be thorough.

Reply:
good point.. it feels like using the blog to document all these features is getting complicated